plot.py

```diff
@@ -603,6 +606,7 @@ def _update_plot(self):
                                                 norm=norm, orientation='horizontal')
         # colorbar - use specified units or default depending on
         # what has or has not been entered
+        #XXX this should be handel else where
         if self.units is None or self.units == '':
```

Comment:
Are you wanting the full path to the units here? I think that it can have the value of None when some file types are read (not containing units or not able to read units).